entrypoint.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+timeout=${SETUP_TIMEOUT:-600}
+additionalDotnetArgs=""
+
+echo "Waiting for $PBURL to be ready..."
+
+n=0
+until [ "$n" -ge "$timeout" ]; do
+  httpStatusCode=$(curl -I -s --insecure $PBURL | cat | head -n 1 | cut -d" " -f2)
```

Comment:
shouldn't all references to $PBURL  be $mpSupplierUrl ?

Reply:
I've left them as is for now, will change them when I get stuck into the Helm bit